### ISSUE TITLE
wiki.vg references

### ISSUE THREAD

**Issue by Timongcraft** [PaperMC/docs#496]
As the site [wiki.vg](https://wiki.vg/) is soon ("November 30th") to be sunset, references should be replaced with their own docs or redirects to a another source.

[wiki.vg](https://wiki.vg/) sunsetting post:
https://tkte.ch/articles/2024/11/11/sunsetting.html

References in the docs (at the time of writing, only 3 references, and 2 distinct ones):
https://github.com/search?q=repo%3APaperMC%2Fdocs+wiki.vg&type=code

At the time of writing, it seems possible that wiki.vg will be merged into minecraft.wiki, according to this page, which might serve as alternative:
https://minecraft.wiki/index.php?title=Minecraft_Wiki:Forum/Wiki.vg_merge

**Comment by Leguan16** [comment]
I think we should wait until we know more. Replacing it now until Minecraft.wiki merged wiki.vg seems like a unnecessary effort I would say.

**Comment by Timongcraft** [comment]
Yeah, absolutely, it shouldn't be rushed, I just wanted to raise awareness because the information didn't seem that widespread.
